scripts/play.py does not work for absolute path

```
audio_common/sound_play/scripts$ ./play.py $(pwd)/../sounds/say-beep.wav 1.0
[INFO] [1653473671.097828]: Playing "/home/k-okada/catkin_ws/ws_audio/src/audio_common/sound_play/scripts/../sounds/say-beep.wav".
```
raise error like
```
process[sound_play/is_speaking-3]: started with pid [982]
[INFO] [1653473766.379425]: Loading from plugin definitions
[INFO] [1653473766.886719]: sound_play node is ready to play sound
[ERROR] [1653473767.063060]: Exception in callback: 'SoundPlayNode' object has no attribute 'filesound'
[INFO] [1653473767.068195]: Traceback (most recent call last):
  File "/home/k-okada/catkin_ws/ws_audio/src/audio_common/sound_play/scripts/soundplay_node.py", line 192, in callback
    sound = self.select_sound(data)
  File "/home/k-okada/catkin_ws/ws_audio/src/audio_common/sound_play/scripts/soundplay_node.py", line 109, in select_sound
    sound = self.filesound[data.arg]
AttributeError: 'SoundPlayNode' object has no attribute 'filesound'
```
which was working on 0.3.13.

--- a/sound_play/scripts/soundplay_node.py
+++ b/sound_play/scripts/soundplay_node.py
@@ -106,7 +106,7 @@ class SoundPlayNode(object):
                             'volume for cached wave has changed,'
                             'resetting volume')
                         filesound.sound.set_property('volume', data.volume)
-                sound = self.filesound[data.arg]
+                sound = self.filesounds[data.arg]
             else:
                 absfilename = os.path.join(
                     roslib.packages.get_pkg_dir(data.arg2), data.arg)
@@ -132,7 +132,7 @@ class SoundPlayNode(object):
                             'volume for cached wave has changed,'
                             'resetting volume')
                         filesound.sound.set_property('volume', data.volume)
-                sound = self.filesound[absfilename]
+                sound = self.filesounds[absfilename]
         elif data.sound == SoundRequest.SAY:
             voice_key = data.arg + '---' + data.arg2
             if voice_key not in self.voicesounds.keys():

